content/en/apps/guides/hosting/4.x/app-developer.md

```diff
@@ -220,6 +220,36 @@ Be sure you want to do this, there is no "are you sure?" prompt and it will dele
 
 Also note that this command will use the `sudo` command when deleting the CouchDB data, so it may prompt for your password.
 
+#### Debugging
+
+To output extra debug information which shows load average, CHT container count, global container count, and a table of services with their status like this:
+
+```
+---DEBUG INFO---
+Load: 3.75 2.92 2.93    
+CHT Containers: 7                                                                                
+Global Containers 15                     
+
+Service              Status   Container                               Image     
+cht-upgrade-service  running  400_deleteme-dir-cht-upgrade-service-1  public.ecr.aws/s5s3h4s7/cht-upgrade-service:latest
+haproxy              NA       NA                                      public.ecr.aws/medic/cht-haproxy:4.4.0-8229-outbound-push
+healthcheck          running  400_deleteme_healthcheck_1              public.ecr.aws/medic/cht-haproxy-healthcheck:4.4.0-8229-outbound-push
+api                  running  400_deleteme_api_1                      public.ecr.aws/medic/cht-api:4.4.0-8229-outbound-push
+sentinel             running  400_deleteme_sentinel_1                 public.ecr.aws/medic/cht-sentinel:4.4.0-8229-outbound-push
+nginx                running  400_deleteme_nginx_1                    public.ecr.aws/medic/cht-nginx:4.4.0-8229-outbound-push
+couchdb              running  400_deleteme_couchdb_1                  public.ecr.aws/medic/cht-couchdb:4.4.0-8229-outbound-push
+```
```

Comment:
I think it makes more sense to document the command you type in and then show the output (ie: swap the order of this section). At the moment you have a sentence that reads "To output extra debug information ... you can prepend the ...", except it doesn't flow because it has a massive code block in the middle.

Reply:
ah - great idea!  Done in e5712030